From 37a03619a179d9146a7eb4470fd9219f3df60558 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Tue, 5 Sep 2023 08:12:33 +0800
Subject: [PATCH] treble-Force-enable-Wireless-Display

---
 system.prop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/system.prop b/system.prop
index f3c0c45..9dde202 100644
--- a/system.prop
+++ b/system.prop
@@ -49,3 +49,6 @@ bluetooth.profile.sap.server.enabled?=true
 
 # Samsung vendors default to a Samsung-specific gadget for MTP. Set this to use AOSP's functionfs MTP
 vendor.usb.use_ffs_mtp=1
+
+# Wireless Display
+persist.debug.wfd.enable=1
-- 
2.34.1

